src/uiSelectSortDirective.js

```diff
@@ -92,6 +92,8 @@ uis.directive('uiSelectSort', ['$timeout', 'uiSelectConfig', 'uiSelectMinErr', f
         }
 
         move.apply(theList, [droppedItemIndex, newIndex]);
+        
+        scope.$parent.$selectMultiple.updateModel();
```

Comment:
I think you can require ^ngModel and not ^uiSelectMultiple and call ngModel.$setViewValue(Date.now());

Is it not the only useful thing that updateModel does here? Everything else seems like overhead in this function for what you are trying to achieve.